tests/llm/test_investigate.py

```
@@ -134,67 +121,33 @@
-        name="Previous Logs", type=SpanTypeAttribute.SCORE
-    ) as logs_span:
-        logs_eval = evaluate_previous_logs_mention(output=output)
-        scores["previous_logs"] = logs_eval.score
-        logs_span.log(
-            scores={
-                "previous_logs": logs_eval.score,
-            },
-            metadata=logs_eval.metadata,
-        )
+    correctness_eval = evaluate_correctness(
+        output=output, expected_elements=expected, parent_span=eval_span
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)